content/practices/landmark-regions/landmark-regions-practice.html

```diff
@@ -252,33 +252,33 @@ <h4>HTML Techniques</h4>
               </li>
             </ul>
 
-            <h4>ARIA Technique</h4>
+            <h4>ARIAテクニック</h4>
 
-            <p>If the HTML <code>footer</code> element technique is not being used, a <code>role=&quot;contentinfo&quot;</code> attribute should be used to define a <code>contentinfo</code> landmark.</p>
+            <p>HTMLの<code>footer</code>要素のテクニックが使用されていない場合、<code>role=&quot;contentinfo&quot;</code>属性を使用して<code>contentinfo</code>ランドマークを定義します。</p>
```

Comment:
```suggestion
            <p>HTMLの<code>footer</code>要素のテクニックが使用されていない場合、<code>role=&quot;contentinfo&quot;</code>属性を使用して<code>contentinfo</code>ランドマークを明示します。</p>
```

Reply:
should be used とありますが、ここでは「するべきです」で訳出しなくてよいですかね・・？

```suggestion
            <p>HTMLの<code>footer</code>要素のテクニックが使用されていない場合、<code>contentinfo</code>ランドマークを明示するために<code>role=&quot;contentinfo&quot;</code>属性を使用するべきです。</p>
```

Reply:
私の見落としですね…修正案でよいと思います！